src/h5web/visualizations/shared/TooltipMesh.tsx

```diff
@@ -32,12 +28,8 @@ function TooltipMesh(props: Props): ReactElement {
   } = useTooltip<Coords>();
 
   // Scales to compute data coordinates from unprojected mesh coordinates
-  const xCoordScale = scaleLinear()
-    .domain([-width / 2, width / 2])
-    .range([0, cols]);
-  const yCoordScale = scaleLinear()
-    .domain([-height / 2, height / 2])
-    .range([0, rows]);
+  const { abscissaScale } = useAbscissaScale();
+  const { ordinateScale } = useOrdinateScale();
```

Comment:
Now using the new hooks to get the scales needed to go from canvas to axis coordinates